Support encrypting .synap models

mdenna-synaptics opened the pull request:
- encrypt-model-ebg now accepts .synap as input
- policy file is now optional, if not specified use default attributes

mdenna-synaptics committed:
Support encrypting .synap models

- encrypt-model-ebg now accepts .synap as input
- policy file is now optional, if not specified use default attributes

ayounes-synaptics reviewed (commented):
Let's wait for proper testing on the target and for Lorenzo's approval as well

mdenna-synaptics committed:
Use replace instead of encrypting in place

mdenna-synaptics committed:
Pad metadata to 16 bytes

- this avoids "check_params : ERROR: Invalid parameter - body size" error message

mdenna-synaptics committed:
Fix padding lenght

mdenna-synaptics commented:
Test successfully
